packages/vkui/src/components/HorizontalCell/HorizontalCell.tsx

```diff
@@ -78,6 +60,11 @@ export interface HorizontalCellProps
    * Актуально для использования в многострочных списках
    */
   noPadding?: boolean;
+  /**
+   * Позволяет поменять тег, используемый для `title`.
+   * По умолчанию Caption для `size=s`, иначе Subhead.
+   */
+  TitleComponent?: React.ElementType;
```

Comment:
```suggestion
  TitleComponent?: React.ComponentType<TypographyProps>;
```
тип нужно сузить, т.к. нам нужен именно типографический компонент, а не просто `h1`, например

или будем позволять любой прокинуть?